reconcile/openshift_resources_base.py

```diff
@@ -154,7 +154,9 @@ def __init__(self, msg):
         )
 
 
-def lookup_vault_secret(path, key, version=None):
+def lookup_vault_secret(path, key, version=None, tvars={}):
```

Comment:
please don't use mutable objects (the dict) as default parameters
this should be:
```
def lookup_vault_secret(path, key, version=None, tvars=None):
    if tvars is None:
        tvars = {}
```